src/routes/adminRoute.js

```diff
@@ -0,0 +1,14 @@
+import express from 'express';
+import Auth from '../middlewares/Auth';
+import checkAction from '../middlewares/roleCheck';
+import adminController from '../controllers/adminContoller';
+
+const admin = express.Router();
+
+const { verifyToken } = Auth;
+
+admin.post('/adminuser', verifyToken, checkAction('User'), adminController.createUser);
```

Comment:
You actually don't need this routes, admin is a user hence user's routes are enough for any admin access and operation